virttest/test_setup.py

```diff
@@ -1387,7 +1402,7 @@ def modprobe_driver(self, driver=None, option=None):
         cmd = "modprobe %s" % driver
         if not option:
             option = self.driver_option
-        if driver and driver != "mlx5_core":
+        if driver and driver != "mlx5_core" and ARCH != 'x86_64':
```

Comment:
earlier this was used by x86_64, as ppc doesn't have any other than mlx, so is it sure that x86_64 never uses it?

Reply:
The option is used to generate vf. Now function `set_vf` does that.  So it can be removed for x86_64 platform.